cmd/skopeo/main.go

```diff
@@ -31,13 +31,15 @@ func createApp() *cli.App {
 			Usage: "enable debug output",
 		},
 		cli.StringFlag{
-			Name:  "cert-path",
-			Value: "",
-			Usage: "use certificates at `PATH` (cert.pem, key.pem) to connect to the registry",
+			Name:   "cert-path",
+			Value:  "",
+			Usage:  "use certificates at `PATH` (cert.pem, key.pem) to connect to the registry",
+			Hidden: true,
 		},
 		cli.BoolTFlag{
-			Name:  "tls-verify",
-			Usage: "require HTTPS and verify certificates when talking to docker registries (defaults to true)",
+			Name:   "tls-verify",
+			Usage:  "require HTTPS and verify certificates when talking to docker registries (defaults to true)",
```

Comment:
If these two are deprecated and don’t do anything, should this say so instead of describing what these options don’t do?  Or perhaps the `Usage` is never used for `Hidden` flags?